api/models/v2/application.go

```diff
@@ -1,11 +1,12 @@
 package models
 
 type Application struct {
-	Name              string             `json:"name" validate:"required"`
-	Description       string             `json:"description"`
-	Homepage          string             `json:"homepage"`
-	Configuration     *ApplicationConfig `json:"config" validate:"dive"`
-	IsProjectAgnostic bool               `json:"is_project_agnostic"`
+	Name                           string                          `json:"name" validate:"required"`
+	Description                    string                          `json:"description"`
+	Homepage                       string                          `json:"homepage"`
+	Configuration                  *ApplicationConfig              `json:"config" validate:"dive"`
+	IsProjectAgnostic              bool                            `json:"is_project_agnostic"`
+	StreamlitPlaceholderPageConfig *StreamlitPlaceholderPageConfig `json:"streamlit_placeholder_page_config"`
```

Comment:
nit: can we use just placeholderpageconfig instead of streamlitplaceholderpageconfig

Reply:
Good point I'll change this :)